### DISCUSSION TITLE
[Question] Accessing best_ckpt path

### DISCUSSION THREAD

**Discussion by austinmw** [open-mmlab/mmengine#625] [Q&A] [answered]
Hi, how can I access the best saved checkpoint path? I tried:

```python
runner.meta['hook_msgs'].get('best_ckpt', None)
```

But got the error:

> AttributeError: 'Runner' object has no attribute 'meta'

**Comment by zhouzaida** [comment]
Hi @austinmw , you could use `runner.message_hub.get_info('best_ckpt')`.

**Comment by HAOCHENYE** [comment]
If you cannot access `runner.message_hub` directly, don't worry, it is a globally accessible instance, you can also get `best_ckpt` by:

```python
MessageHub.get_current_instance().get_info('best_ckpt')
```

**Comment by HAOCHENYE** [comment]
@austinmw Hi! I thought other users might also be interested in this question :laughing:, and I convert it to a discussion.

**Comment by austinmw** [comment, marked as answer]
Thanks!
